src/modules/dashboard/components/DomainDropdown/DomainSelectItem.tsx

```diff
@@ -0,0 +1,108 @@
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+import React, {
+  KeyboardEventHandler,
+  MouseEventHandler,
+  useCallback,
+} from 'react';
+
+import Button from '~core/Button';
+import ColorTag, { Color } from '~core/ColorTag';
+import Heading from '~core/Heading';
+import Icon from '~core/Icon';
+import Paragraph from '~core/Paragraph';
+import { ColonyDomainsQuery } from '~data/index';
+import { ENTER } from '~types/index';
+
+import styles from './DomainSelectItem.css';
+
+interface Props {
+  domain: ColonyDomainsQuery['colony']['domains'][number];
```

Comment:
Maybe we should have a "proper' type for a domain? Something like `AnyDomain` ?

Reply:
Ah yes, nice call! There's a `OneDomain` type exported from `data` 🙂